src/Operator.ts

```diff
@@ -1,7 +1,7 @@
 import {Subscriber} from './Subscriber';
 
 export class Operator<T, R> {
-  call<T, R>(subscriber: Subscriber<R>): Subscriber<T> {
+  call<R>(subscriber: Subscriber<R>): Subscriber<T> {
```

Comment:
Good catch; that's incorrect. `R` should be bound to the type's `R` as well. I think fixing this will greatly expand the scope of operators that need to be touched however. I'll try it out.